### PR TITLE
Feature/api

### PR DESCRIPTION
**Pull request by sashameniakin**
added fetch API, context, some cards

### CODE REVIEW

File: pages/funds/index.js [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+
+  const setData = useCallback(async () => {
+    try {
+      let apiResponse = await getCoins();
+      setCoinData(apiResponse);
+    } catch (error) {
+      console.log(error.message);
+    }
+  }, [getCoins]);
+  console.log(coinData);
```
Comment (stefankorder) [inline pages/funds/index.js:23]: Please delete this console log.

File: pages/funds/index.js [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+  }, []);
+
+  const setData = useCallback(async () => {
+    try {
+      let apiResponse = await getCoins();
+      setCoinData(apiResponse);
+    } catch (error) {
+      console.log(error.message);
+    }
+  }, [getCoins]);
```
Comment (stefankorder) [inline pages/funds/index.js:22]: If you put this part inside the useEffect, you won't need to disable eslint.

File: pages/funds/index.js [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+import styled from "styled-components";
+
 export default function Funds() {
-  return <p>Page Funds is under construction</p>;
+  let {getCoins} = useContext(CMContext);
+  let [coinData, setCoinData] = useState(null);
+
+  useEffect(() => {
+    setData();
+    /* eslint-disable-next-line react-hooks/exhaustive-deps */
```
Comment (stefankorder) [inline pages/funds/index.js:12]: Take a look at my comment below.

File: .eslintrc.json [hunk 4]
```diff
@@ -10,4 +10,7 @@
     "next",
     "next/core-web-vitals"
   ]
+  /*  "rules": {
+    "react-hooks/exhaustive-deps": off,
+  } */
```
Comment (stefankorder) [inline .eslintrc.json:15]: You really need this unused code?

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Take a look at my comments.

File: pages/funds/index.js (hunk 1 above)
Comment (sashameniakin) [reply pages/funds/index.js:23]: done!

File: pages/funds/index.js (hunk 2 above)
Comment (sashameniakin) [reply pages/funds/index.js:22]: useEffect does not allow you to add async in the callback function, as google said

File: .eslintrc.json (hunk 4 above)
Comment (sashameniakin) [reply .eslintrc.json:15]: Oh, no, deleted, thank you

File: pages/funds/index.js (hunk 2 above)
Comment (stefankorder) [reply pages/funds/index.js:22]: You can not directly change the callback function to a async function put you can put an async function inside the callback function of the useeffect.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Take a look at my comment.

**Review by stefankorder — APPROVED** [review]
LGTM!

**Pull request merged**